src/dsm/headers/Node.hpp

```diff
@@ -225,6 +225,12 @@ namespace dsm {
     std::optional<std::pair<Delay, Delay>> m_delay;
     Delay m_counter;
     Delay m_phase;
+    /// @brief Variabile che mi dice come sono in rapporto tra di loro il tempo di verde e il tempo di rosso
```

Comment:
I got the point of this variable but I miss the point of such implementation.
It should take values in the set {0, 1, 2} but is initialized at 7.
Moreover, its settings are all hard-coded so it's very weird...

Given that this implementation is not directly related to the new optimization algorithm, I propose to open an issue which aims to add this mechanism in order to re-think it in a general way.
What do you guys think @sbaldu @vero-dav-vero?